api-audit/src/main/java/com/capitalone/dashboard/evaluator/RegressionTestResultEvaluator.java

```diff
@@ -62,6 +62,10 @@
     private static final String FAILURE_COUNT = "failureCount";
     private static final String SKIP_COUNT = "skippedCount";
     private static final String TOTAL_COUNT = "totalCount";
```

Comment:
The audit api changes need to be in a separate PR

Reply:
It is moved to separate PR